src/browser/browser.js

```diff
@@ -101,12 +102,18 @@ module.exports = class Browser {
     }
 
     _startCollectingCustomCommands() {
-        this._session.overwriteCommand("addCommand", (origCommand, name, ...rest) => {
-            if (!this._session[name]) {
-                this._customCommands.add(name);
+        const browserCommands = getBrowserCommands();
+        const elementCommands = getElementCommands();
+
+        this._session.overwriteCommand("addCommand", (origCommand, name, wrapper, elementScope) => {
```

Comment:
I need `elementScope` field in order to understand how exactly I should add command in browser environment.